Resources/public/css/styles.css

```diff
@@ -496,3 +496,15 @@ div.sonata-filters-box div.form-group span.input-group-addon {
     padding: 3px 10px;
     font-size: 13px;
 }
+
+.hide {
+    display: none;
```

Comment:
@OskarStark I implement remove empty blocks, but on layout still trash DIVs, which extended from sonata_block.templates.block_base. I don't know how to delete it.